python/tank/pipelineconfig_factory.py

```diff
@@ -600,6 +583,35 @@ def _get_pipeline_configs_for_path(path, data):
     return all_matching_pcs
 
 
+def _add_to_project_paths(project_paths, project_name, s, pc):
+
```

Comment:
docstring. also please don't call variables `s` and `pc` - something more descriptive would be great